src/components/Tables/Table.cy.tsx

```diff
@@ -225,11 +275,55 @@ describe("<Table />", () => {
     });
 
     it("check all rows ticks the checkall box", () => {
-        cy.mount(<Table data={smallerData} headers={headers} />);
+        cy.mount(
+            <ThemeProvider theme={mockTheme}>
+                <Table data={smallerData} headerKeysAndLabels={headers} checkboxes={true} />
+            </ThemeProvider>
+        );
         cy.get("input[aria-label='Select row 0']").click();
         cy.get("input[aria-label='Select row 1']").click();
         cy.get("input[aria-label='Select row 2']").click();
 
         cy.get("input[aria-label='Select all rows']").should("be.checked");
     });
+
+    it("checkbox disabling works", () => {
+        cy.mount(
+            <ThemeProvider theme={mockTheme}>
+                <Table data={smallerData} headerKeysAndLabels={headers} checkboxes={false} />
+            </ThemeProvider>
+        );
+        cy.get("input[aria-label='Select row 0']").should("not.exist");
+        cy.get("input[aria-label='Select row 5']").should("not.exist");
+        cy.get("input[aria-label='Select row 10']").should("not.exist");
+    });
+
+    it("accordion works", () => {
+        cy.mount(
+            <ThemeProvider theme={mockTheme}>
+                <Table
+                    data={smallerData}
+                    headerKeysAndLabels={headers}
+                    checkboxes={false}
+                    toggleableHeaders={["full_name", "phone_number"]}
+                />
+            </ThemeProvider>
+        );
+
+        cy.get("div.MuiAccordionSummary-root").click();
+        cy.get("div.MuiAccordionDetails-root").contains("Name");
+        cy.get("div.MuiAccordionDetails-root").contains("Phone Number");
+    });
+
+    it("filter input renders", () => {
+        cy.mount(
+            <ThemeProvider theme={mockTheme}>
+                <Table data={smallerData} headerKeysAndLabels={headers} checkboxes={false} />
+            </ThemeProvider>
+        );
+
+        cy.get("input[placeholder*='Filter by Name']").should("exist");
+        cy.get("input[placeholder*='Filter by Phone Number']").should("exist");
+        cy.get("div.MuiAccordionSummary-root").contains("Select Columns");
+    });
```

Comment:
I'd rather leave themeing tests until the new themes have been merged.

Reply:
I'd rather you make 'some' that work on current main, but with the intention that they will be changed 'trivially' by converting the `const mockTheme` into a new format and changing the test a tiny amount, rather than the theme person (Chloe?) having to remember to write a test and having to know all the context